src/umbraco.cms/businesslogic/translation/Translation.cs

```diff
@@ -19,13 +19,15 @@ public class Translation
         public static void MakeNew(CMSNode Node, User User, User Translator, Language Language, string Comment,
                                    bool IncludeSubpages, bool SendEmail)
         {
+            // Get translation taskType for obsolete task constructor
+            var taskType = ApplicationContext.Current.Services.TaskService.GetTaskTypeById(1);

```

Comment:
Try to avoid hardcoding ids, better to use `GetTaskTypeByAlias("toTranslate")` here, can you update please?